api/models/node.go

```diff
@@ -112,12 +123,14 @@ type IndexAutoCompleteTagValues struct {
 }
 
 func (t IndexAutoCompleteTagValues) Trace(span opentracing.Span) {
-	span.SetTag("org", t.OrgId)
-	span.SetTag("Prefix", t.Prefix)
-	span.SetTag("tag", t.Tag)
-	span.SetTag("expressions", t.Expr)
-	span.SetTag("from", t.From)
-	span.SetTag("limit", t.Limit)
+	span.SetTag("orgId", t.OrgId)
```

Comment:
Let's be consistent and use lowercase orgid everywhere

Reply:
I agree with consistency, but that would be to use `orgId` everywhere, which we are doing except in 1 place.